components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/consent/ConsentMgtPostAuthnHandler.java

```diff
@@ -1176,6 +634,22 @@ private String getStandardDialect(AuthenticationContext context) {
         }
         return null;
     }
+
+    @Override
+    public int getPriority() {
+
+        return 110;
```

Comment:
Please no hard coding :). We already have a post handler with fixed priority. Hence others need to be configurable around it.

Reply:
+1. The handler is already enabled in the identity.xml with an order id.